platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/style/expressions/Expression.java

```diff
@@ -3701,6 +3750,21 @@ private static Expression convert(@NonNull JsonPrimitive jsonPrimitive) {
         throw new RuntimeException("Unsupported literal expression conversion for " + jsonPrimitive.getClass());
       }
     }
+
+    /**
+     * Converts a raw expression to a DSL equivalent.
+     *
+     * @param rawExpression the raw expression to convert
+     * @return the resulting expression
+     * @see <a href="https://www.mapbox.com/mapbox-gl-js/style-spec/">Style specification</a>
+     */
+    public static Expression convert(@NonNull String rawExpression) {
+      return convert(getGsonInstance().fromJson(rawExpression, JsonArray.class));
+    }
+
+    private static Gson getGsonInstance() {
+      return gson = gson != null ? gson : new Gson();
```

Comment:
what is the need for lazy initialising? Think we can directly initialise it in the field without too much overhead.